factory/ideas.php

```diff
@@ -148,11 +148,19 @@ public function get_ideas($number = 10, $sort = 'date', $sort_direction = 'DESC'
 			$where .= ' AND i.idea_votes_up > i.idea_votes_down';
 		}
 
+		// Only get approved topics
+		$where .= ' AND t.topic_visibility = ' . ITEM_APPROVED;
```

Comment:
Just making sure: this means that moderators would need to go through the MCP to approve pending posts, right?

Reply:
I think so. That should not be changing how it currently behaves.